modules/admin_manual/pages/enterprise/monitoring/index.adoc

```diff
@@ -0,0 +1,4 @@
+:section-title: Enterprise Monitoring
+:section-preamble-ender: to monitor the ownCloud Enterprise installation 
```

Comment:
I like the idea, but should be approved by @pmaier1.

Reply:
The Metrics app is rather a **reporting** tool than a monitoring tool. Monitoring has a more technical connotation (load, response times, etc.), while the metrics in here rather tells you something about the usage of the installation. Please change this in a follow-up PR.

Reply:
ok, takes only a view minutes

Reply:
PR made, see #3225